docs/adr/0053-web-socket-data-synchronization.md

```diff
@@ -0,0 +1,21 @@
+# 53. Web socket data synchronization
+
+Date: 2024-11-10
+
+## Status
+
+Accepted
+
+## Context
+
+- Currently Twake Mail web use Firebase Cloud Messaging to sync data on real time
+- JMAP already implemented web socket push, which is more optimized for web
+
+## Decision
+
+- Web socket is implemented for real time update data for Twake Mail web
+- Service worker is implemented for background tasks, helping web socket working in background
```

Comment:
we should not mention service worker anymore

Reply:
Done